sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -1124,4 +1125,15 @@ String getLink() {
     ItemDeserializer getItemDeserializer() {
         return getDatabase().getDocClientWrapper().getItemDeserializer();
     }
+
+    /**
+     * Obtains a list of {@link FeedRange} that can be used to parallelize Feed
+     * operations.
+     *
+     * @return An unmodifiable list of {@link FeedRange}
+     */
+    @Beta(Beta.SinceVersion.V4_9_0)
+    public Mono<List<FeedRange>> getFeedRanges() {
```

Comment:
Shouldn't this be `Flux<FeedRange>` or `CosmosPagedFlux<FeedRange>` ? What is the benefit of `Mono<List<FeedRange>>` - as that will always have just one list of FeedRange ?

Reply:
Yes - will always be just one returned list - never pages or multiple requests/IO operations

Reply:
So intentionally using Mono<List<T>> - Flux would inidcate that there could be multiple asynchronous values